stmhal/boards/STM32F429DISC/pins.csv

```diff
@@ -0,0 +1,38 @@
+SW,PA0
+PA0,PA0
+PA5,PA5
+PA8,PA8
+PA9,PA9
+PA10,PA10
+PA13,PA13
+PA14,PA14
```

Comment:
My STM32F429-Discovery board has PA0 thru PA15 exposed.
Same for PB0-15, PC0-15, PD0-15, PE0-15, PF0-15, PG0-15, PH0-1

Every pin on the connector should be included here, since you don't know how the user is going to use their particular board.

I also see PC0-15